### PR TITLE
Focus #input_to when forwarding

### LINKED ISSUE #2941: After clicking "Forward" the "Recipients" input field should be focused
### Steps to reproduce

1. Open any encrypted message
2. Click "Forward"

### Actual behavior 

The focus is in `#input_text`

### Expected behavior 

The focus is in `#input_to`

### PR DESCRIPTION
**Pull request by limonte**
Fixes #2941

### CODE REVIEW

File: extension/chrome/elements/compose-modules/compose-render-module.ts [hunk 1]
```diff
@@ -24,18 +24,20 @@ import { KeyUtil } from '../../../js/common/core/crypto/key.js';
 
 export class ComposeRenderModule extends ViewModule<ComposeView> {
 
-  public renderReplyMsgComposeTable = async (method: 'forward' | 'reply' = 'reply'): Promise<void> => {
+  private replyBoxMethod!: 'reply' | 'forward';
```
Comment (tomholub) [inline extension/chrome/elements/compose-modules/compose-render-module.ts:27]: We could consider `responseMethod` or maybe `responseReachIntention`.. just trying to think

File: extension/chrome/elements/compose-modules/compose-quote-module.ts [hunk 2]
```diff
@@ -196,7 +196,6 @@ export class ComposeQuoteModule extends ViewModule<ComposeView> {
     $(el).remove();
     Xss.sanitizeAppend(this.view.S.cached('input_text'), this.getTripleDotSanitizedFormattedHtmlContent());
     this.tripleDotSanitizedHtmlContent = undefined;
-    this.view.inputModule.squire.focus();
```
Comment (tomholub) [inline extension/chrome/elements/compose-modules/compose-quote-module.ts:199]: As long as the result is that body gets focused on reply and inputs get focused on forward, it's all good :+1:

**Review by tomholub — DISMISSED** [review]
Code looks good :)

File: extension/chrome/elements/compose-modules/compose-render-module.ts (hunk 1 above)
Comment (limonte) [reply extension/chrome/elements/compose-modules/compose-render-module.ts:27]: `responseMethod` look good 👍

File: extension/chrome/elements/compose-modules/compose-quote-module.ts (hunk 2 above)
Comment (limonte) [reply extension/chrome/elements/compose-modules/compose-quote-module.ts:199]: All is well

> body gets focused on reply and inputs get focused on forward

this is handled in https://github.com/FlowCrypt/flowcrypt-browser/pull/2985/files#diff-107b333d89353f1f2b84b7959666d8ccR251

File: test/source/tests/page-recipe/compose-page-recipe.ts [hunk 3]
```diff
@@ -85,9 +85,6 @@ export class ComposePageRecipe extends PageRecipe {
   }
 
   public static fillRecipients = async (composePageOrFrame: Controllable, recipients: Recipients, windowType: 'new' | 'reply') => {
-    if (windowType === 'reply') { // new messages should already have cc/bcc buttons visible, because they should have recipients in focus
-      await composePageOrFrame.waitAndClick('@action-show-container-cc-bcc-buttons');
```
Comment (limonte) [inline test/source/tests/page-recipe/compose-page-recipe.ts:89]: Focus is already in `@input-to`, no need to click the placeholder `@action-show-container-cc-bcc-buttons` in test.

**Review by tomholub — APPROVED** [review]

**Review by tomholub — APPROVED** [review]

**Pull request merged**